client/src/components/squids/SquidList.jsx

```diff
@@ -33,6 +70,33 @@ export const SquidList = () => {
     <div className="squids-list">
       <h2 className="squids-list__header">Squid List</h2>
       {squidListQueryOutput}
+      <div className="squids-pagination">
+        <Link
+          to="/squids?page=1"
```

Comment:
I would link to `/squids` and my hope is that it would function the same as `/squids?page=1`